scripts/git/common.ps1

```diff
@@ -55,9 +55,9 @@ function Get-StagedFiles {
 
 # Formats the given file using ClangFormat
 function ClangFormatFile([string]$repoRoot, [string]$filename, [bool]$isDryRun){
-    $clangCommand = "$repoRoot/tools/scripts/clang-format.ps1"
+    $clangCommand = "$repoRoot/scripts/git/clang-format.ps1"
```

Comment:
This branch changed the location of the scripts without updating the scripts, needed to fix this ro get my pre-commit to run